include/fmt/chrono.h

```diff
@@ -46,8 +46,8 @@ FMT_BEGIN_NAMESPACE
 #  if FMT_HAS_INCLUDE("winapifamily.h")
 #    include <winapifamily.h>
 #  endif
-#  if defined(_WIN32) && (!defined(WINAPI_FAMILY) || \
-                          (WINAPI_FAMILY == WINAPI_FAMILY_DESKTOP_APP))
+#  if defined(_WIN32) && !FMT_WIN9X && \
```

Comment:
Typo here and below:
```suggestion

#  if defined(_WIN32) && !FMT_WIN98X && \
```

Clearly needs CI coverage 😁

Reply:
Good catch, thanks, fixed! (`FMT_WIN9X` now used throughout)

Reply:
You are most welcome.